src/Spice86.Core/Emulator/OperatingSystem/VirtualDeviceBase.cs

```diff
@@ -0,0 +1,15 @@
+namespace Spice86.Core.Emulator.OperatingSystem;
+
+public abstract class VirtualDeviceBase : IVirtualDevice {
+    protected VirtualDeviceBase(DeviceAttributes attributes, ushort strategy = 0, ushort interrupt = 0) {
+        Attributes = attributes;
+        StrategyEntryPoint = strategy;
+        InterruptEntryPoint = interrupt;
+    }
+
+    public ushort Segment { get; set; }
+    public ushort Offset { get; set; }
+    public DeviceAttributes Attributes { get; set; }
+    public ushort StrategyEntryPoint { get; set; }
```

Comment:
What is StrategyEntyPoint ? Can it be documented in a XML summary ?

Reply:
I documented them.

They're not used yet. This is for stuff to do with INT 21 44 (ioctl)